File: PHP/7php 2/sponge_meme.php
<?php
/*
|--------------------------------------------------------------------------
| sPoNgEbOb MeMe
|--------------------------------------------------------------------------
|
*/

function sponge_meme($sentence) {
   $res = "";
   for ($i = 0; $i < strlen($sentence); $i++) {
      $res .= $i%2==0 ? strtoupper($sentence[$i]) : strtolower($sentence[$i]);
   }
   return $res;
}


function sponge_meme(string $sentence): string {
   return join(array_map(fn($x,$k)=>$k&1?strtolower($x):strtoupper($x),str_split($sentence),range(0,strlen($sentence))));
}


function sponge_meme(string $sentence): string {
   $res = str_split($sentence);
   foreach($res as $key => $value){
      $res[$key] = $key%2==0 ? strtoupper($value) : strtolower($value);
   };
   return implode('', $res);
}


print_r(sponge_meme("stop Making spongebob Memes!") . " StOp mAkInG SpOnGeBoB MeMeS!");

File: Python/7py 3/sponge_meme.py
"""
|--------------------------------------------------------------------------
| sPoNgEbOb MeMe
|--------------------------------------------------------------------------
|
"""

def sponge_meme(s):
   return ''.join([s[i].upper() if i%2==0 else s[i].lower() for i in range(len(s))])


def sponge_meme(sentence):
   res = ""
   for i in range(len(sentence)):
      res += sentence[i].upper() if i % 2 == 0 else sentence[i].lower()
   return res


def sponge_meme( s ):
   return "".join([x.upper(), x.lower()][i % 2] for i, x in enumerate(s))


print(sponge_meme("stop Making spongebob Memes!"), "StOp mAkInG SpOnGeBoB MeMeS!")